centaur/src/main/resources/standardTestCases/google_labels/bad_options.json

```diff
@@ -0,0 +1,5 @@
+{
+  "google_labels": {
+    "custom-label-label-label-label-label-label-label": "0custom-value"
```

Comment:
Is this key intended to run afoul of a length limit? There does not appear to be an assertion for it, i.e. the test that uses this file fails for a different reason

Reply:
Now asserted